src/app/accounts/login.component.ts

```diff
@@ -120,6 +121,20 @@ export class LoginComponent extends BaseLoginComponent implements OnDestroy {
                 this.successRoute = this.redirectUri;
             }
         });
+
+        // Cozy custo
+        // send an event to Pass Addon to get user's password in order to autologin.
+        document.addEventListener('cozy.passwordextension.credentials.response', (evt: any) => {
+            this.forceLoading = true;
+            this.masterPassword = evt.detail;
+            // submit only after a short delay in order to avoid a blinking screen for the user
+            setTimeout(() => {
+                this.forceLoading = false;
+                this.submit();
+            }, 1100);
```

Comment:
Does this 1100ms value means something? Is this arbitrary?

Reply:
Without timeout the login screen flashes rapidly and unpleasantly. 
The chosen timeout avoids this flickering and allows the user to understand that the addon's authentication service is working for him. It's empirical

Reply:
Ok, it would be great to add this info in a comment near the timeout.